docs/modules/druid/partials/nav.adoc

```diff
@@ -1,8 +1,14 @@
 * xref:druid:getting_started/index.adoc[]
 ** xref:druid:getting_started/installation.adoc[]
 ** xref:druid:getting_started/first_steps.adoc[]
+* xref:druid:usage-guide/index.adoc[]
+** xref:druid:usage-guide/pod-placement.adoc[]
+** xref:druid:usage-guide/ingestion.adoc[]
+** xref:druid:usage-guide/deep-storage.adoc[]
+** xref:druid:usage-guide/resources-and-storage.adoc[]
+** xref:druid:usage-guide/security.adoc[]
+** xref:druid:usage-guide/logging.adoc[]
+** xref:druid:usage-guide/monitoring.adoc[]
+** xref:druid:usage-guide/configuration-and-environment-overrides.adoc[]
```

Comment:
Hmm, I thought about it too to have it in the same section, and I think that might be a good place. The whole page needs an overhaul as well though, it's not very useful the way it is now.

If it were to move into "usage guide", then basically everything is in there, so we could even get rid of "usage guide" alltogether. Then the menu would be flatter.

Reply:
I think the page should be called differently (more like "meta config" or something) because it is about configuring the operator itself, instead of ocnfiguring a druid cluster. For the same reason I think it also makes sense to keep it in a different section. It's just that it's not a well written page for now